src/js/components/AppListItemComponent.jsx

```diff
@@ -89,9 +90,11 @@ var AppListItemComponent = React.createClass({
 
     let labelsWidth = 0;
     let numberOfVisibleLabels = 0;
+    let labelNodes = React.findDOMNode(refs.labels).querySelectorAll(".label");
 
-    refs.labels.props.children[0].find((label) => {
-      labelsWidth += DOMUtil.getOuterWidth(React.findDOMNode(refs[label.ref]));
+    // labelNodes is not an Array, but a NodeList
+    [].forEach.call(labelNodes, function (label) {
```

Comment:
Should we introduce an own forEach-function into DOMUtil, like described in this article (recommendations at bottom)?
What do you think?
http://toddmotto.com/ditch-the-array-foreach-call-nodelist-hack/